tool/speed.cc

```diff
@@ -1907,6 +1966,7 @@ bool Speed(const std::vector<std::string> &args) {
      !SpeedECDH(selected) ||
      !SpeedECDSA(selected) ||
      !SpeedECKeyGen(selected) ||
+     !SpeedDilithium(selected) ||
```

Comment:
This is going to have to go in the !OpenSSL block down by SpeedKEM, and the above methods should go inside the if/def next to the KEM benchmark.

Reply:
Addressed in [096f384](https://github.com/awslabs/aws-lc/pull/724/commits/096f3840d3a43a74655b995a630e0544ae9b9afd), let me know if I misunderstood.